contracts/deploy/077_balancer_rETH_WETH_2.js

```diff
@@ -0,0 +1,65 @@
+const { deploymentWithGovernanceProposal } = require("../utils/deploy");
+const addresses = require("../utils/addresses");
+const { balancer_rETH_WETH_PID } = require("../utils/constants");
+
+const platformAddress = addresses.mainnet.rETH_WETH_BPT;
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "077_balancer_rETH_WETH_2",
```

Comment:
Re: The suffix `_2` isn't necessary, right? (since the migration number is prepended)